service/src/main/java/bio/terra/pipelines/service/QuotasService.java

```diff
@@ -64,11 +64,13 @@ public UserQuota getQuotaForUserAndPipeline(String userId, PipelinesEnum pipelin
    * @return - the updated user quota object
    */
   public UserQuota updateQuotaConsumed(UserQuota userQuota, int newQuotaConsumed) {
-    if (newQuotaConsumed < 0) {
-      throw new InternalServerErrorException("Quota consumed must be greater than or equal to 0");
-    }
-    if (newQuotaConsumed > userQuota.getQuota()) {
-      throw new InternalServerErrorException("Quota consumed cannot be greater than user quota");
+    if (newQuotaConsumed <= 0 || newQuotaConsumed > userQuota.getQuota()) {
+      logger.error(
+          "Issue with updating quota consumed: User quota: {}, current quota consumed: {}, new quota consumed {}",
+          userQuota.getQuota(),
+          userQuota.getQuotaConsumed(),
+          newQuotaConsumed);
+      throw new InternalServerErrorException("Internal Error");
```

Comment:
cool this is nice, this is how we want it to work. it's possible that in https://broadworkbench.atlassian.net/browse/TSPS-340 we could do this at a higher level but this is good for now